docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:3.5
+LABEL maintainer Jan Koppe <post@jankoppe.de>
+
+ARG PUMPIO__GUID=1337
+ARG PUMPIO__UID=1337
+
+ENV PUMP_LOCATION="/usr/local/pumpio"
+ENV PUMP_GIT_TAG="v4.0.0-beta.5"
+ENV PUMP_GIT_REPO="https://github.com/pump-io/pump.io.git"
+
+RUN apk add --no-cache graphicsmagick openssl nodejs python make g++ git \
+     && mkdir -p "${PUMP_LOCATION}" \
+     && git clone --recursive --depth 1 --branch "${PUMP_GIT_TAG}" "${PUMP_GIT_REPO}" "${PUMP_LOCATION}" \
```

Comment:
As I mentioned in #1307 it seems better/more intuitive to me to build the Docker image from the local source instead of cloning. Is this not how it's usually done? You seem to know more than me soooo...